src/utils.py

```
@@ -8,11 +8,29 @@
 from supabase import create_client, Client
 from urllib.parse import urlparse
 import openai
+
+try:
+    import google.generativeai as genai
+except Exception:  # noqa: PIE786
+    genai = None
 import re
```

Ruff (0.11.9)

`re` imported but unused

Remove unused import: `re`

(F401)